bin/pasta_show_maintainers.py

```diff
@@ -0,0 +1,299 @@
+            for maintainer in flatten(maintainers):
+                if not (type(maintainer) == str 
+                        or maintainer[0] == '' 
+                        or maintainer[1] == ''):
+                    name = pretty_name(maintainer)
+                    loc, byte = file_sizes[filename]
+                    if name in counts:
+                        maintainer_to_subsystem[name].add(subsystem)
+                        counts[name].increase(filename, loc, byte,
+                                filter_by_files and filename in filter_by_files)
+                    else:
+                        maintainer_to_subsystem[name] = set([subsystem])
+                        counts[name] = CountItem(filename, loc, byte,
+                                filter_by_files and filename in filter_by_files)
+                elif verbose:
+                    log.info('Ignoring mailing list:'+ str(maintainer))
+    if filter_by_files:
+        # If filter is given, only return the relevant lines:
+        counts = {name:item for name,item in counts.items() 
+                    if item.display_in_filter()}      
+    # Sort maintainers by decreasing total line counts
+    return decreasing_order(counts), maintainer_to_subsystem
+
+def show_maintainers(config, sub, argv):
+    parser = ArgumentParser(description='Display file sizes grouped by ' 
+                                            'maintainers or subsystems')
+    parser.add_argument('show_maintainers', nargs=1, type=bool)
+    parser.add_argument('--entries', action='store_true', 
+                            help='groups by subsystems')
+    parser.add_argument('--verbose', action='store_true', 
+                            help='Show ignored exceptions on the way')
+    parser.add_argument('--smallstat', action='store_true', help='Simple view')
+    parser.add_argument('--bytes', action='store_true', help='Show byte counts')
+    parser.add_argument('--outfile', type=str, help='Output to a csv file')
+    parser.add_argument('--file', type=str, help='Specify MAINTAINERS file')
+    parser.add_argument('--filter', type=str, help='Filter by file list')
+    parser.add_argument('--commit', type=str, 
+                            help='Specify a commit hash for Linux repo')
+    parser.add_argument('--cpu', dest='cpu_factor', metavar='cpu', type=float,
+                        default=1.0, help='CPU factor for parallelisation '
+                                          '(default: %(default)s)')
+    args = parser.parse_args()
+    d_repo = config.repo_location
+    repo = pygit2.Repository(d_repo)
+    commit_version = args.commit if args.commit else 'HEAD'
+    commit = repo.revparse_single(commit_version)
+    tree = commit.tree
+        
+    if args.filter:
+        filter_by_files = file_to_string(args.filter).splitlines()
+        # Automatically find kernel version from file name "v5.6"
+        version_regex = re.compile(r'v[0-9]+.[0-9]+')
```

Comment:
Simply split the filename at the dots. Then, there's no need for a regex (that does, for example, currently not work for v5.4-rc3).